### DISCUSSION TITLE
feature suggestion: commenting on a particular file

### DISCUSSION THREAD

**Discussion by victorel-petrovich** [community/community#62535] [Repositories]
It seems to me that it would be useful to have, on the page where a file is displayed for reading (after navigating to it from `<>Code` tab): 
the ability for users to comment on that particular file.
Perhaps similar to how it's possible to comment on a particular commit.

But maybe nicer if the commenting section was not under the file, but besides it (hide-able) so that one could always scroll the code to be in line with the comment one is making/reading.

**Comment by Livia-Rasp** [comment]
What you can do is, if you go into the Blame view of a file:
You can select a line and reference it in an issue. So if you find something to change about a file in your repository, you could handle it that way.
Maybe this helps you for what you're wanting to do.

**Reply by victorel-petrovich** [reply]
Thank you for the tip, I'm going to use that as a workaround! It will help.

But I still think it would be more convenient to comment besides the file. Together with same ability to reference  particular line(s) there, so that when you click on the referenced lines in the comment - that would make the code automatically scroll to line up ! 

 So I'm going to leave this suggestion open, to see what others think.
